src/components/fields/image-upload/image-input/file-item/file-item.tsx

```diff
@@ -141,32 +143,70 @@ export const FileItem = ({ id = "file-item", index, fileItem, maxSizeInKb, accep
 		);
 	};
 
-	return (
-		<Wrapper
-			isError={isError}
-			id={TestHelper.generateId(`${id}-${index + 1}`)}
-			data-testid={TestHelper.generateId(`${id}-${index + 1}`)}
-		>
+	const renderCellInfoDetails = () => {
+		// Helper function to generate IDs and testIDs
+		const generateTestIdAndId = (prefix, suffix) => TestHelper.generateId(`${prefix}-${suffix}`, "image");
```

Comment:
- this is causing all the elements to have the same id and same testids, instead of `image`
-  it should differ between elements 
- noticed the same prefix and suffix used for all the elements, you can hard code that instead